src/mscorlib/src/System/HResults.cs

```diff
@@ -79,157 +74,65 @@ internal static class HResults
+        internal const int COR_E_SAFEHANDLEMISSINGATTRIBUTE = unchecked((int)0x80131623);
         internal const int COR_E_SECURITY = unchecked((int)0x8013150A);
+        internal const int COR_E_SEMAPHOREFULL = unchecked((int)0x8013152B);
         internal const int COR_E_SERIALIZATION = unchecked((int)0x8013150C);
-        internal const int COR_E_SERVER = unchecked((int)0x8013150e);
         internal const int COR_E_STACKOVERFLOW = unchecked((int)0x800703E9);
         internal const int COR_E_SYNCHRONIZATIONLOCK = unchecked((int)0x80131518);
         internal const int COR_E_SYSTEM = unchecked((int)0x80131501);
         internal const int COR_E_TARGET = unchecked((int)0x80131603);
         internal const int COR_E_TARGETINVOCATION = unchecked((int)0x80131604);
-        internal const int COR_E_TARGETPARAMCOUNT = unchecked((int)0x8002000e);
+        internal const int COR_E_TARGETPARAMCOUNT = unchecked((int)0x8002000E);
         internal const int COR_E_THREADABORTED = unchecked((int)0x80131530);
         internal const int COR_E_THREADINTERRUPTED = unchecked((int)0x80131519);
         internal const int COR_E_THREADSTART = unchecked((int)0x80131525);
         internal const int COR_E_THREADSTATE = unchecked((int)0x80131520);
+        internal const int COR_E_THREADSTOP = unchecked((int)0x80131521);
         internal const int COR_E_TIMEOUT = unchecked((int)0x80131505);
         internal const int COR_E_TYPEACCESS = unchecked((int)0x80131543);
         internal const int COR_E_TYPEINITIALIZATION = unchecked((int)0x80131534);
         internal const int COR_E_TYPELOAD = unchecked((int)0x80131522);
         internal const int COR_E_TYPEUNLOADED = unchecked((int)0x80131013);
         internal const int COR_E_UNAUTHORIZEDACCESS = unchecked((int)0x80070005);
+        internal const int COR_E_UNSUPPORTEDFORMAT = unchecked((int)0x80131523);
         internal const int COR_E_VERIFICATION = unchecked((int)0x8013150D);
         internal const int COR_E_WAITHANDLECANNOTBEOPENED = unchecked((int)0x8013152C);
         internal const int CORSEC_E_CRYPTO = unchecked((int)0x80131430);
         internal const int CORSEC_E_CRYPTO_UNEX_OPER = unchecked((int)0x80131431);
-        internal const int CORSEC_E_INVALID_IMAGE_FORMAT = unchecked((int)0x8013141d);
-        internal const int CORSEC_E_INVALID_PUBLICKEY = unchecked((int)0x8013141e);
-        internal const int CORSEC_E_INVALID_STRONGNAME = unchecked((int)0x8013141a);
         internal const int CORSEC_E_MIN_GRANT_FAIL = unchecked((int)0x80131417);
-        internal const int CORSEC_E_MISSING_STRONGNAME = unchecked((int)0x8013141b);
         internal const int CORSEC_E_NO_EXEC_PERM = unchecked((int)0x80131418);
         internal const int CORSEC_E_POLICY_EXCEPTION = unchecked((int)0x80131416);
-        internal const int CORSEC_E_SIGNATURE_MISMATCH = unchecked((int)0x80131420);
         internal const int CORSEC_E_XMLSYNTAX = unchecked((int)0x80131419);
```

Comment:
Some of these look unused, like `CORSEC_E_XMLSYNTAX`. It can be nice to have follow up cleanup.

Reply:
It is used in CoreRT in [this file](https://github.com/dotnet/corert/blob/master/src/System.Private.Interop/src/WinRT/ExceptionHelpers.cs#L628) along with many others CoreCLR does not have. When I share the file in a later change, I will have to include them.

Reply:
I think S.P.Interop in CoreRT should keep own non-shared copy of the file.

Reply:
It may be useful to mark the eventual shared HResults as partial class so that different runtimes, different OSes, or different libraries can add to it own local codes.